app/controllers/v1/health_controller.rb

```diff
@@ -0,0 +1,15 @@
+module V1
+  class HealthController < ApplicationController
+    respond_to :json
```

Comment:
Can we take this out and just `render :json` in the show method? I get a missing template if I try to hit id/v1/status (without a .json or accept header) in a browser.